me.kozec.syncthingtk.yaml

```diff
@@ -41,27 +41,46 @@ modules:
       - /share/man
 
   - name: syncthing
+    disabled: false
     buildsystem: simple
+    build-options:
+        env:
+            GOBIN: /app/bin
+        #build-args:
+        #    - --share=network
+
     build-commands:
       - |
         . /usr/lib/sdk/golang/enable.sh
         export GOPATH=$PWD/go
-        cd go/src/github.com/syncthing/syncthing
-        go run build.go -no-upgrade
-      - install -Dm755 -t /app/bin go/src/github.com/syncthing/syncthing/bin/syncthing
+        go version
+        go mod vendor
+        #go build -mod=vendor
+        env GOFLAGS=-mod=vendor go run build.go -no-upgrade build
+      - install -Dm755 -t /app/bin syncthing
     sources:
+      - generated-sources.json
+      #- type: git
```

Comment:
Using binary releases shouldn’t really ever be used in this Flatpak, I think we can let go of this completely.

Reply:
I agree that the binary release shouldn't be used. But it's not a "no-go" in flatpak land. I was tempted to use the binary release to not have the hassle with building it from source and someone in the future might rather want to use the binary directly. But yeah, it's a left over and can be discarded.

Reply:
I have removed the reference.